src/seedu/addressbook/AddressBook.java

```diff
@@ -206,6 +206,11 @@
      * ====================================================================
      */
 
+    /**
+     * Main entry point of the application.
+     * If a valid path to a storage file is provided as an argument, that file
+     * will be loaded instead of the default.
```

Comment:
Is it important to note down how the main() function handles command-line arguments here? How would this approach scale to an application which handles many more command-line arguments? Would we have to note them all down in the javadoc? That would be hard to maintain, I would imagine.